user-guide/Troubleshooting/Known_issues/KI_MySQL_Unhandled_Exception.md

```diff
@@ -0,0 +1,44 @@
+---
+uid: KI_MySQL_Unhandled_Exception
+---
+
+# MySQL.Data.dll can cause service crashes
```

Comment:
Could you rephrase this title so it is more a description of the issue than of the cause? With the current title, people who go looking for the issue based on what they encounter in their system may not know that this is the page they are looking for.

Reply:
Will do.